ipn/ipnlocal/local.go

```diff
@@ -3304,20 +3304,31 @@ func (b *LocalBackend) isLocalIP(ip netip.Addr) bool {
 }
 
 var (
-	magicDNSIP   = tsaddr.TailscaleServiceIP()
-	magicDNSIPv6 = tsaddr.TailscaleServiceIPv6()
+	serviceIP   = tsaddr.TailscaleServiceIP()
```

Comment:
Maybe there's a better name of this than "serviceIP"?

Reply:
I'd prefer we omit this rename in this PR, as we try to keep changes to be cherry picked as small as possible.